packages/block-library/src/latest-posts/index.php

```diff
@@ -5,6 +5,13 @@
  * @package WordPress
  */
 
+$block_core_latest_posts_excerpt = 0;
+
+function get_block_core_latest_posts_excerpt() {
```

Comment:
I think we need to add documentation to this function so it can be part of core.

Reply:
Added PHPDoc to both new things. Thanks!